app/client/components/dashboard/beneficiaryLiveFeed.jsx

```diff
@@ -69,13 +71,45 @@ class BeneficiaryLiveFeed extends React.Component {
                               </UserTime>
                             );
 
-                            if (transfer.transfer_type === 'PAYMENT') {
+                            let currency, fromExchangeTransfer, recipientCurrency;
+                            const transferAccountId = transfer.sender_transfer_account_id;
+                            if (transferAccountId) {
+                              const transferAccount = this.props.transferAccounts.byId[transferAccountId];
+                              currency = transferAccount && transferAccount.token && transferAccount.token.symbol;
```

Comment:
we should use this syntax more. it's sooo clleaaan

Reply:
yeah I really like `currency = transferAccount && transferAccount.token && transferAccount.token.symbol;`... feels like python.

@wangfowen's code :)